### DISCUSSION TITLE
DigitalOcean credit error

### DISCUSSION THREAD

**Discussion by sxg024** [community/community#145657] [GitHub Education]
### Hi everyone,

I'm trying to connect my github account to DigitalOcean to get the $200 in platform credit for 1 year perk, but I keep getting the error "The GitHub Student Pack promotion has already been applied to another team. Please switch to the team it was applied to use your credits."
Can you help please?
Thanks.

**Comment by davevad93** [comment]
Hi @sxg024 ,

I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/education) select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue with DigitalOcean.
